### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by lhvk**
Made the whole page: index, profile, login

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css");
+/* ========================================================
+VARIABLES
+======================================================== */
+/* ========================================================
+NAVIGATION
+======================================================== */
+.navbar,
+.offcanvas {
+  background-color: #1e3163;
```
Comment (Fermain) [inline css/styles.css:13]: Colours should be variables

File: index.html [hunk 2]
```diff
@@ -7,7 +7,184 @@
     <link
       rel="stylesheet"
       href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
-    <title>Document</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
```
Comment (Fermain) [inline index.html]: You should not be using bootstrap CDN for styles

File: index.html [hunk 3]
```diff
@@ -7,7 +7,184 @@
...
+          tabindex="-1"
+          id="offcanvasNavbar"
+          aria-labelledby="offcanvasNavbarLabel">
+          <div class="offcanvas-header">
+            <h5 class="offcanvas-title" id="offcanvasNavbarLabel">
+              Social Site
+            </h5>
+            <span
+              class="bi bi-x navbar-toggler border-0 p-0"
+              style="color: #fff; font-size: 2rem"
```
Comment (Fermain) [inline index.html]: Do not use inline styles. Make a custom class instead.

File: index.html [hunk 4]
```diff
@@ -7,7 +7,184 @@
...
+        </div>
+      </div>
+    </nav>
+    <!-- Navigation end -->
+
+    <!-- Feed & card container start-->
+    <div class="container mt-3">
+      <!-- Feed heading start -->
+      <div class="row justify-content-center">
+        <div class="col" style="max-width: 720px">
```
Comment (Fermain) [inline index.html]: Inline styles. Use col-md-6 etc to control sizing.

File: index.html [hunk 5]
```diff
@@ -7,7 +7,184 @@
...
+    <!-- Feed & card container end -->
+    <!-- Footer start -->
+    <footer><div>2022 &copy;</div></footer>
+    <!-- Footer end -->
+    <!-- Bootstrap JS -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js"
+      integrity="sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3"
+      crossorigin="anonymous"></script>
+    <script
```
Comment (Fermain) [inline index.html]: If you use bootstrap.bundle you don't need two imports here.

File: package.json [hunk 6]
```diff
@@ -20,7 +20,8 @@
   },
   "homepage": "https://github.com/lhvk/lars-halvor-kallak-css-frameworks-ca#readme",
   "dependencies": {
-    "bootstrap": "^5.2.2"
+    "bootstrap": "^5.2.2",
+    "bootstrap-icons": "^1.9.1"
```
Comment (Fermain) [inline package.json:24]: Load these from node_modules instead of CDN

File: scss/styles.scss [hunk 7]
```diff
@@ -0,0 +1,28 @@
+/* ========================================================
+IMPORTS
+======================================================== */
+
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css");
+@import "_variables.scss";
+@import "_nav.scss";
+@import "_card.scss";
+@import "_button.scss";
```
Comment (Fermain) [inline scss/styles.scss:9]: You do not import Bootstrap.scss. This is a hard requirement for this assignment

File: index.html (hunk 3 above)
Comment (lhvk) [reply index.html]: Replaced inline styles with custom css styles

File: index.html (hunk 4 above)
Comment (lhvk) [reply index.html]: Used col-classes to adjust width

File: scss/styles.scss (hunk 7 above)
Comment (lhvk) [reply scss/styles.scss:9]: This file? \node_modules\bootstrap\scss\bootstrap.scss

File: package.json (hunk 6 above)
Comment (lhvk) [reply package.json:24]: How do I do that?

File: index.html (hunk 5 above)
Comment (lhvk) [reply index.html]: Done. Imported bundle instead.

File: index.html (hunk 2 above)
Comment (lhvk) [reply index.html]: Removed CDN from head on all html.

File: package.json (hunk 6 above)
Comment (Fermain) [reply package.json:24]: You need to import `bootstrap.scss` from node_modules into your main SCSS file.

In your HTML you will link to `/dist/css/styles.css` which is the output from SASS.

File: scss/styles.scss (hunk 7 above)
Comment (Fermain) [reply scss/styles.scss:9]: That's correct

**Pull request merged**